biglittleian-hometown.md

```diff
@@ -0,0 +1,11 @@
+## Cupertino
+
+### Great places to eat (that are affordable)
+- dumpster behind Alexanders
+- ketchup packets from Wendy's (avoid eye contact with teller)
+- Rio Adobe
```

Comment:
@biglittleian I :heart: the focus on affordable - any affordable Italian places?